index.js

```diff
@@ -41,7 +41,7 @@ function lro(options) {
   options = extend({
     scopes: lro.ALL_SCOPES
   }, options);
-  var gaxGrpc = gax.grpc(options);
+  var gaxGrpc = grpc(options);
```

Comment:
`gax` is referred below in this file (line 55-57), so it's still good to be there.